components/ads-client/src/test_utils.rs

```diff
@@ -0,0 +1,146 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+* License, v. 2.0. If a copy of the MPL was not distributed with this
+* file, You can obtain one at http://mozilla.org/MPL/2.0/.
+*/
+
+#[cfg(test)]
```

Comment:
you can probably just wrap the "mod test_utils" statement in lib.rs to make the entire module conditional on tests, meaning you can remove these individual cfg checks?